guides/get_action_context.md

```diff
@@ -0,0 +1,50 @@
+# Get action context information 
+
+If you have registered a Client action handler callback for your App, then when your App is called through this callback, it will receive an action handler context token. Your App needs to use this token with this API to extract the proper action context.
+
+### Path
+```
+GET /v1/apps/{appId}/getActionContext/{actionHandlerContextToken}
+```
+actionHandlerContextToken has only 5 minutes liftime, and can only be used once. It will be deleted once it is used. 
+
+#### Parameters
+
+|Type|Name|Description|Schema|
+|---|---|---|---|
+|**Header**|Authorization required|Authorization header, in form of Bearer {access_token}. The access_token is issued as result of an oauth flow by Watson Work Services|String|
+|**Path**|appId required|Unique id of an app|String|
+|**Path**|actionHandlerContextToken required|token generated by caller before|String|
+
+#### Responses
+
+|HTTP Code|Description|Schema|
+|---|---|---|
+|**200**|The configuration data is returned and deleted successfully.|ConfigurationData(see below)|
+|**401**|Unauthorized.||
+|**403**|Forbidden.||
+|**404**|Not Found. The app with the given `appId` or the configuration data for the given `configurationToken` could not be found.||
+|**500**|Internal server error.||
+
+#### Produces
+
+* `application/json`
+
+#### ActionContext
+
+App configuration specific case is triggered via open URL (configure URL provided by app).
+The table below list the fields that actionContext contains for app configuration.
+
+|Name|Description|Schema|
+|---|---|---|
+|**type**  <br>*required*|The action type for users to be aware of the configuration context. It should be either space-app-config-requested or team-app-config-requested.|string|
+|**appId**  <br>*optional*|Id of the app for which the configuration data has been generated.|string|
+|**userId**  <br>*required*|Id of the user by which the configuration data has been generated.|string|
+|**actioncontext**  <br>*required*|The action context structure according to the action type.|JSONObject|
+
+- type: space-app-config-requested
```

Comment:
Not sure describing the types of actions and their context belongs in this doc.  It should refer to either a dedicated listing of all action Types with nicely described context or explicit explanation of each in dedicated guide docs.  @jarusso   This is not unlike how we described all the types of webhook events

Reply:
@miguelestrada @jarusso Revised this page for action types based on Wei Hua's suggestion. Please review and see if the page meets the requirement. Thanks!